pkg/log/emperror_test.go

```diff
@@ -0,0 +1,39 @@
+package log_test
```

Comment:
How are we testing things here? :)

Reply:
Yeah, wanted to bring that up. We're not testing anything. But it's a 'structure' for manual verification? Not sureif we should just dump it or what to do with it

Reply:
Ok lets keep it like that but maybe follow-up with https://github.com/maistra/istio-workspace/issues/833 should be considered as good time of evaluating it. WDYT?

Reply:
That sounds good